adobecom/milo
# Quiz Results Debugging

- Adds debugging functionality to quickly test different quiz result pages and build sharable urls.
- Removes query parameter updating and cleans up that unnecessary code.

Resolves: [MWPW-125569](https://jira.corp.adobe.com/browse/MWPW-125569)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/delta/app-recommender-test/?martech=off
- After: https://quiz-results-debug--milo--fullcolorcoder.hlx.page/drafts/delta/app-recommender-test/?debug-results&martech=off

To Verify: 
- Visit [quiz](https://quiz-results-debug--milo--fullcolorcoder.hlx.page/drafts/delta/app-recommender-test/?debug-results), take quiz, view results page.
- Open clean incognito browser (or clear local storage) and hit paste. 
- Should redirect to results page with identical content as before.
